wcivf/apps/elections/templates/elections/postcode_view.html

```diff
@@ -92,7 +92,7 @@ <h2>{{ postelection.friendly_name }}</h2>
               {{ postelection.post.label }} ward.
               {% endif %}
 
-              {% if postelection.winner_count %}
+              {% if postelection.winner_count and postelection.election.voting_system_id == 'FPTP' %}
               You will have {{ postelection.winner_count|intword }} vote{{ postelection.winner_count|pluralize }}.
```

Comment:
I'm just removing this for everything which isn't definitely FPTP (i.e: we know its something else, or `voting_system` is unknown) for now. We can probably do some better stuff here if we wanted. Do note though that populating the `voting_system` key here does make this code: https://github.com/DemocracyClub/WhoCanIVoteFor/blob/3c01c12e1bd1d7f7e9bc2f44e3bf95451c13221c/wcivf/apps/elections/templates/elections/postcode_view.html#L139-L147 work (which it didn't before), and that does provide some more general information about the voting system further down the page.